frontend/src/reducers/accessRequestReducer.tsx

```diff
@@ -30,7 +30,7 @@ export interface IAccessRequestState {
 }
 
 const initialState: IAccessRequestState = {
-  pagedAccessRequests: { page: 0, total: 0, quantity: 0, items: [] },
+  pagedAccessRequests: { page: 1, pageIndex: 0, total: 0, quantity: 0, items: [] },
```

Comment:
I'm not loving that we need to keep track of "two" page instances (one for frontend and one for server). I'm not sure why you need the server page in the frontend as it can be calculated: `page = pageIndex + 1` just before issuing an API request.

This is no blocker and will likely be revisited when we refactor the tables